dump/mapping/bufr_satwnd_amv_ahi_mapping.yaml

```diff
@@ -0,0 +1,158 @@
+
+    satelliteId:
+      query: "*/SAID"
+
+    satelliteZenithAngle:
+      query: "*/SAZA"
+
+    sensorCentralFrequency:
+      query: "*/SCCF"
+
+    pressure:
+      query: "*/PRLC"
+
+    # Processing Center
+    dataProviderOrigin:
+      query: '*/OGCE'
+    
+    # Generating Application - contains multiple types
+    generatingApplication:
+      query: '*/QCPRMS[1]/GNAP'
+
+    # Quality Information - contains multiple types
+    qualityInformation:
+      query: '*/QCPRMS[1]/PCCF'
+
+    # Wind Retrieval Method Information - Computation 
+    windComputationMethod:
+      query: '*/SWCM'
+
+    # ObsValue - Wind Direction
+    windDirection:
+      query: '*/WDIR'
+
+    # ObsValue - Wind Speed
+    windSpeed:
+      query: '*/WSPD'
+
+  splits:
+    satId:
+      category:
+        variable: satelliteId
+        map:
+          _173: h8 
+          _174: h9  
+
+encoder:
+# type: netcdf
+
+# dimensions:
+
+  globals:
+    - name: "platformCommonName"
+      type: string
+      value: "Himawari"
+
+    - name: "platformLongDescription"
+      type: string
+      value: "Geostationary Operational Satellite"
+
+    - name: "sensor"
+      type: string
+      value: "Advanced Himawari Imager"
+
+    - name: "source"
+      type: string
+      value: "NCEP BUFR Dump for satellite derived atmospheric motion vectors (satwnd)"
+
+    - name: "providerFullName"
+      type: string
+      value: "Japan Meteorological Agency"
+
+    - name: "processingLevel"
+      type: string
+      value: "Level-2"
+
+    - name: "converter"
+      type: string
+      value: "BUFR"
+
+  variables:
+
+    # MetaData 
+    - name: "MetaData/dateTime"
+      source: variables/timestamp
+      longName: "Datetime"
+      units: "seconds since 1970-01-01T00:00:00Z"
+
+    - name: "MetaData/latitude"
+      source: variables/latitude
+      longName: "Latitude"
+      units: "degrees_north"
+      range: [ -90, 90 ]
+
+    - name: "MetaData/longitude"
+      source: variables/longitude
+      longName: "Longitude"
+      units: "degrees_east"
+      range: [ -180, 180 ]
+
+    - name: "MetaData/satelliteIdentifier"
+      source: variables/satelliteId
+      longName: "Satellite Identifier"
+
+    - name: "MetaData/satelliteZenithAngle"
+      source: variables/satelliteZenithAngle
+      longName: "Satellite Zenith Angle"
+      units: "degree"
+      range: [ 0, 90 ]
+
+    - name: "MetaData/sensorCentralFrequency"
+      source: variables/sensorCentralFrequency
+      longName: "Satellite Channel Center Frequency"
+      units: "Hz"
+
+    - name: "MetaData/dataProviderOrigin"
+      source: variables/dataProviderOrigin
+      longName: "Identification of Originating/Generating Center"
+
+    - name: "MetaData/windComputationMethod"
+      source: variables/windComputationMethod
+      longName: "Satellite-derived Wind Computation Method"
+
+    - name: "MetaData/pressure"
+      source: variables/pressure
+      longName: "Pressure"
+      units: "pa"
```

Comment:
I think this should be "Pa" instead of lowercase

Reply:
Nice catch!!  Thanks.  Fixed.